src/components/Messages/Message/styles.scss

```diff
@@ -0,0 +1,30 @@
+@import '../../../styles/colors';
+@import '../../../styles/variables';
+
+.surveyButton {
+	&__content {
+		display: flex;
+
+		flex-direction: row;
+
+		width: 200px;
+
+		margin: 8px auto;
+		padding: 10px;
+
+		color: black;
+
+		border: 1px solid #abdbe3;
```

Comment:
```suggestion
```

We don't need to add border with same color as background color

Reply:
Border color is inherited by the button component, so need to override